### PR TITLE
@cbrown/1191 refactor gitea

### PR DESCRIPTION
**Pull request by Collinbrown95**
Add json 6902 patches for required istio annotations to allow traffic to postgres database in init container. Part of the solution to https://github.com/StatCan/daaas/issues/1191.

Add initial README.md content

### CODE REVIEW

File: daaas-system/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Deployment Overview
+
+The primary tool used for AAW deployments is [ArgoCD]().
```
Comment (cboin1996) [inline daaas-system/README.md:3]: Are you missing a link to argocd here?

**Review by cboin1996 — CHANGES_REQUESTED** [review]
Almost LGTM, I think you need to update the `securityContext` of the init container as per point 1 in issue https://github.com/StatCan/daaas/issues/1191. 
Referring to the upstream helm chart for gitea, I found https://gitea.com/gitea/helm-chart/src/tag/v5.0.4/values.yaml#L22 as a way to configure `securityContext` globally for all containers. I dont think this is what you would want however, so I propose a kustomize patch to change the `securityContext` to `1337` in the manifest [here](https://github.com/StatCan/aaw-argocd-manifests/blob/aaw-dev-cc-00/profiles-argocd-system/template/gitea/protected-b/manifest.yaml#L421)

File: daaas-system/README.md (hunk 1 above)
Comment (Collinbrown95) [reply daaas-system/README.md:3]: Good catch re: `securityContext`! I forgot to add this. The latter is what we want here (i.e. update the `securityContext` just for the `configure-gitea` init container).

Also yes - I was just going to link to https://argo-cd.readthedocs.io/en/stable/ (for readers who don't know our deployment infrastructure).

**Review by cboin1996 — APPROVED** [review]
Lgtm

**Pull request merged**